common/util/FileUtil.cpp

```diff
@@ -66,6 +79,14 @@ std::string get_project_path() {
 }
 
 std::string get_file_path(const std::vector<std::string>& input) {
+  // TODO - clean this behaviour up, it causes unexpected behaviour when working with files
+  // the project path should be explicitly provided by whatever if needed
+  // TEMP HACK
+  // - if the provided path is absolute, don't add the project path
+  if (input.size() == 1 && std::filesystem::path(input.at(0)).is_absolute()) {

```

Comment:
should we make this a different function, instead of having it do different things with absolute and relative paths?

I'm not opposed to this (it makes sense to let absolute paths be absolute paths), but I'm just wondering.

Reply:
So the main reason I had to make this change is this method is used by the file-stream stuff:
https://github.com/open-goal/jak-project/blob/da00b1ff133488afffe651d5c5964d206e70e38f/game/sce/sif_ee.cpp#L100-L102

I think long-term yeah we probably want a more explicit function like `get_project_file_path` and that function linked above should not be assuming everything is within the project directory.  I think we can do a lot of cleanup / simplification, but im not sure how much we want to do immediately.